src/backend/BSXML.pm

```diff
@@ -513,6 +513,7 @@ our $sourceinfolist = [
 our $buildinfo = [
     'buildinfo' =>
 	'project',
+        'config',
```

Comment:
Could you please move that down a bit so that it comes somewhere after the `[]` line? The build config tends to be quite big and contains newlines, so we want an element and not an attribute here.

Sorry for not catching this before. The rest is fine.

Reply:
I moved `config` down, just before `debuginfo`. I hope it is fine there.